### DISCUSSION TITLE
Feature Request: Layer focus / selection via Transform Hierarchy view?

### DISCUSSION THREAD

**Discussion by CBrosM** [keiwando/toonsquid#325] [Ideas]
When animating a lot of attribute keyframes it becomes a bit cumbersome to quickly find the correct layer. Because the transform hierarchy window only shows layer names, it’s a lot quicker to scroll and find the layer you need - plus more screen real estate; If there was the ability to double-click on the name, and it then pulled focus in the main layer menu to continue animating, that would be a great improvement.

**Comment by Cat-Butler** [comment]
Somewhat limited tip: if you have the timeline showing and tap on another layer on the canvas with the Transform tool (not always possible, but works if the layer is currently visible to tap upon), the timeline will autoscroll to the selected layer.

**Reply by CBrosM** [reply]
Thanks, and this does work at times - but it’s often more of a hindrance than a help i.e. inadvertently selecting a layer I don’t want to when animating in the current layer : )
